.gitignore

```diff
@@ -193,6 +193,7 @@ sapi/phpdbg/phpdbg_parser.h
 sapi/phpdbg/phpdbg
 scripts/php-config
 scripts/phpize
+scripts/phpize.m4
```

Comment:
Hello, thanks for suggesting this patch. The `scripts/phpize.m4` file is actually required when building PHP because it will be part of the phpize process for installing custom extensions by the user. So this line shouldn't be part of this pull request.

Reply:
The with this change, file is created by the `configure` script, because it needs to insert the prefix and suffix constants at the right places. See configure.ac:1536 and the file was renamed to a version with `.in` containing the replaceable strings. Otherwise the extension creations scripts will seek `program_prefix` and `program_suffix` from their own `configure` invocation.

Feel free to test a clean build with and without the program-prefix/suffix options from this code. It works.

Reply:
Ok, let me check this out.

Reply:
Correct, got it.. :)